openshift/distro/ubi/9/Dockerfile

```diff
@@ -26,22 +26,19 @@
 #                   with number of cpus being a good heuristic.
 #
 
-FROM registry.access.redhat.com/ubi8/ubi-minimal
+FROM registry.access.redhat.com/ubi9/ubi-minimal
 
 ENV HOME=/home
 WORKDIR "${HOME}/app"
 
-ARG RUBY_VERSION="3.0"
-ARG RUNTIME_DEPS="ruby"
+ARG RUBY_VERSION="3.3"
 
-RUN echo -e "[ruby]\nname=ruby\nstream=${RUBY_VERSION}\nprofiles=\nstate=enabled\n" > /etc/dnf/modules.d/ruby.module \
- && microdnf update --nodocs \
- && microdnf install --nodocs ${RUNTIME_DEPS} \
+RUN microdnf module enable -y "ruby:${RUBY_VERSION}" \
```

Comment:
you can use `microdnf` to enable the `ruby` module, so there's no need to install it manually. This change it's not crucial because it worked with the previous code, but I think it's more readable.